src/sentry/models/notificationaction.py

```diff
@@ -134,14 +134,18 @@ class Meta:
 class ActionTrigger(FlexibleIntEnum):
     """
     The possible sources of action notifications.
-    Use values less than 100 here to avoid conflicts with getsentry's trigger values.
+    Items prefixed with 'GS_' are have registrations in getsentry.
     """
 
     AUDIT_LOG = 0
+    GS_SPIKE_PROTECTION = 100
```

Comment:
Only registering spike protection since the spend allocations value was unused anyway. Registering it with the same int value should mean even though the registrations are made in getsentry with a different (and now unused) enum, they should still map to the same actions and not conflict.